app/screens/options_modal/options_modal.js

```diff
@@ -45,6 +45,10 @@ export default class OptionsModal extends PureComponent {
         };
     }
 
+    shouldComponentUpdate(nextProps) {
```

Comment:
What would cause this to rerender other than a change in dimensions?

Reply:
nothing else that I'm aware of, I'll try to make this a PureComponent and I'll see how many times it re-renders when changing the dimensions on the device